### DISCUSSION TITLE
Open Verified Person (OpenVP) Credential

### DISCUSSION THREAD

**Discussion by amanji** [bcgov/bc-wallet-mobile#506] [Use Cases]
#### Unrevocable
* Site: https://openvp-candy-issuer-dev.apps.silver.devops.gov.bc.ca/
* ledger: [CANdy-Dev](https://candyscan.idlab.org/home/CANDY_DEV)
* [DID/NYM](https://candyscan.idlab.org/tx/CANDY_DEV/domain/14) (Ui6HA36FvN83cEtmYYHxrn)
* [Schema ](https://candyscan.idlab.org/tx/CANDY_DEV/domain/15)(Ui6HA36FvN83cEtmYYHxrn:2:unverified_person:0.1.0)
* [Credential Definition](https://github.com/bcgov/bc-wallet-mobile/discussions/370) (Ui6HA36FvN83cEtmYYHxrn:3:CL:15:default)
#### Revocable

* Site: https://openvp-rev-issuer-dev.apps.silver.devops.gov.bc.ca/credential-data
#### Unrevocable
Site: https://openvp-candy-issuer-test.apps.silver.devops.gov.bc.ca/
* ledger: [CANdy-Test](https://candyscan.idlab.org/home/CANDY_TEST)
* [DID/NYM](https://candyscan.idlab.org/tx/CANDY_TEST/domain/36) (HTkhhCW1bAXWnxC1u3YVoa)
* [Schema ](https://candyscan.idlab.org/tx/CANDY_TEST/domain/38)(HTkhhCW1bAXWnxC1u3YVoa:2:unverified_person:0.1.0)
* [Credential Definition](https://candyscan.idlab.org/tx/CANDY_TEST/domain/39) (HTkhhCW1bAXWnxC1u3YVoa:3:CL:38:default)

#### Revocable
* Site: https://openvp-rev-issuer-test.apps.silver.devops.gov.bc.ca/credential-data

#### "PROD"
Site: https://openvp-candy.vonx.io/
* ledger: [CANdy-Test](https://candyscan.idlab.org/home/CANDY_TEST)
* [DID/NYM](https://candyscan.idlab.org/tx/CANDY_TEST/domain/40) (4eCXHS79ykiMv2PoBxPK23)
* [Schema](https://candyscan.idlab.org/tx/CANDY_TEST/domain/42) (4eCXHS79ykiMv2PoBxPK23:2:unverified_person:0.1.0)
* [Credential Definition](https://candyscan.idlab.org/tx/CANDY_TEST/domain/43)(4eCXHS79ykiMv2PoBxPK23:3:CL:42:default)

#### TEST
Site: https://openvp-candy-issuer-test.apps.silver.devops.gov.bc.ca/
* ledger: [CANdy-Dev](https://candyscan.idlab.org/home/CANDY_DEV)
* [DID/NYM](https://candyscan.idlab.org/tx/CANDY_DEV/domain/19) (XZQpyaFa9hBUdJXfKHUvVg)
* [Schema ](https://candyscan.idlab.org/tx/CANDY_DEV/domain/20)(XZQpyaFa9hBUdJXfKHUvVg:2:unverified_person:0.1.0)
* [Credential Definition](https://candyscan.idlab.org/tx/CANDY_DEV/domain/21) (XZQpyaFa9hBUdJXfKHUvVg:3:CL:20:default)

#### "PROD"
Site: https://openvp-candy.vonx.io/
* ledger: [CANdy-Dev](https://candyscan.idlab.org/home/CANDY_DEV)
* [DID/NYM](https://candyscan.idlab.org/tx/CANDY_DEV/domain/22) (9wVuYYDEDtpZ6CYMqSiWop)
* [Schema](https://candyscan.idlab.org/tx/CANDY_DEV/domain/23) (9wVuYYDEDtpZ6CYMqSiWop:2:unverified_person:0.1.0)
* [Credential Definition](https://candyscan.idlab.org/tx/CANDY_DEV/domain/24)(9wVuYYDEDtpZ6CYMqSiWop:3:CL:23:default)

**Comment by WadeBarnes** [comment]
The `test` and `"prod"` CANdy - Unverified Person Issuers (OpenVP Issuers) have been migrated to the proper ledgers and are now wired to the BC Endorser Services.  Tracking issue: https://github.com/bcgov/essential-services-delivery/issues/124.

Note: The Open Verified Person (OpenVP) Credential is meant for testing purposes only.  Such DIDs and services do not get rooted to production ledgers, so the `"prod"` instance is rooted on CANdy-Test.

The environment documentation above has been updated to reflect the changes.

**Comment by WadeBarnes** [comment]
The `"prod"` URL has changed from `https://openvp-candy-dev.vonx.io/` to `https://openvp-candy.vonx.io/` to reflect the credentials issued are no longer rooted on CANdy-Dev.  The documentation above has been updated to reflect this change.
